_sass/articles/quickstart.scss

```diff
@@ -0,0 +1,78 @@
+.btn-group.btn-group-quickstart {
+  display: flex;
+  margin: $lg 0;
+
+  @include media(md) {
+    flex-wrap: wrap;
+  }
+
+  @include media(sm) {
+    flex-direction: column;
+  }
+
+  .btn {
+    flex: 1 1 50%;
+    height: 72px;
+
+    @include media(sm) {
+      margin-left: 0;
+      margin-top: -$input-border;
+    }
+
+    &:first-child {
+      border-radius: $input-border-radius-lg 0 0 $input-border-radius-lg;
+
+      @include media(md) {
+        border-radius: $input-border-radius-lg 0 0;
+        margin-left: 0;
+      }
+
+      @include media(sm) {
+        border-radius: $input-border-radius-lg $input-border-radius-lg 0 0;
+      }
+    }
+
+    &:nth-child(2) {
+
+      @include media(md) {
+        border-radius: 0 $input-border-radius-lg 0 0;
+      }
+
+      @include media(sm) {
+        border-radius: 0;
+      }
+    }
+
+    &:nth-child(3) {
+
+      @include media(md) {
+        border-radius: 0 0 0 $input-border-radius-lg;
+        margin-left: 0;
+        margin-top: -$input-border;
+      }
+
+      @include media(sm) {
+        border-radius: 0;
+      }
+    }
+
+    &:last-child {
```

Comment:
Use the same selector for this too `nth-child(4)`, since you're selecting the 4th element, not just any last element.